webpack.config.js

```diff
@@ -14,14 +65,12 @@ module.exports = {
   },
   devtool: "source-map",
   resolve: {
-    extensions: ['.ts', '.js'],
-    plugins: [new TsconfigPathsPlugin()],
-    alias: {
-      '@': path.resolve('src')
-    }
+    extensions: ['.ts', '.js', '.proto'],
+    plugins: [new TsconfigPathsPlugin()]
   },
-  node: {
-    fs: 'empty'
+  target:'node',
+  externals: {
+    fs: "commonjs fs"
```

Comment:
What does `fs: "commonjs fs"` mean?

Reply:
This tells webpack that `fs` is a library that is not resolved by webpack, but are dependencies of the output (i.e. dependencies of `dist/polykey.js`). The use of `commonjs` here just tells webpack what the module context is.

This probably needs some extra care to make it isomorphic like `EncryptedFS` is. For Electron, this is not an issue with the following code:
```
app.on('ready', () => {
    mainWindow = new BrowserWindow({
        webPreferences: {
            nodeIntegration: true, // <--- flag
        }
    });
});
```